taichi/transforms/offload.cpp

```diff
@@ -85,19 +85,35 @@ class Offloader {
         } else {
           offloaded->block_dim = s->block_dim;
         }
-        if (auto val = s->begin->cast<ConstStmt>()) {
+
+        // TODO: We need to update codegen for each backend gradually so let's
```

Comment:
When we merge this in let's also notify people working on the vulkan ndarray this change